src/Sylius/Behat/Behaviour/ChoosesCode.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Behaviour;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+trait ChoosesCode
```

Comment:
We have it implemented already. And this is `ChoosesName` trait, as it chooses names